can/interfaces/socketcan/socketcan.py

```diff
@@ -21,6 +21,29 @@
 log_tx = log.getChild("tx")
 log_rx = log.getChild("rx")
 
+can_config_lib = ctypes.util.find_library("socketcan")
```

Comment:
Simpler to use this library rather than parsing output from `ip` but adds a dependency.  Thoughts?